tests/src/main/scala/ch/renku/acceptancetests/ReplShell.scala

```diff
@@ -0,0 +1,26 @@
+package ch.renku.acceptancetests
+
+import org.openqa.selenium.{By, WebDriver}
+import org.openqa.selenium.chrome.ChromeOptions
+import org.openqa.selenium.remote.RemoteWebDriver
+import org.scalatestplus.selenium.{Chrome, Driver, WebBrowser}
+
+/**
+  * A shell for interactively controlling the browser.
+  */
+object ReplShell {
```

Comment:
Is this object just to try something out? If yes, maybe it doesn't have to go into the PR?

Reply:
I improved the code and added some documentation to explain how this can be used. Please take a look again.